services/API-service/src/scripts/mock.service.ts

```diff
@@ -189,10 +190,24 @@ export class MockService {
         }
 
         if (this.shouldMockAlertPerLeadTime(disasterType)) {
+          if (MOCK_USE_OLD_PIPELINE_UPLOAD) {
```

Comment:
@jannisvisser should this IF have an ELSE block?

Reply:
@gulfaraz yes, the part below it should be in else. I guess I wanted to go for an 'early return' type of coding here, but that doesn't work if you're not actually returning/continuing from something to skip the code below it.. Can you change it?